CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing to Talos’ client
+
+Talos’ client follows a certain set of guidelines. This is to ensure that the developers can keep up with the rapid changes towards the beginning of the project, and to be able to maintain certain code standards (and otherwise) further down the line.
+
+## Getting started
+
+- Pull the repository
+- Sign up for a [ZenHub](https://www.zenhub.com/) account
+
+## Setup
+
+`cd` into the project
+
+Copy the contents of `env.example` into `.env` like so:
```

Comment:
what's this for?

Reply:
Here, just read [this](https://medium.com/@ktruong008/absolute-imports-with-create-react-app-4338fbca7e3d).

Reply:
cool, I like that but I've read before that you shouldn't commit the .env file so is this a hacky way to bypass that?

Reply:
I’ll answer here as well: the `.env` file isn’t committed. Take a closer look. We’re copying the contents of `.env.example` into `.env` and `.env` is in our `.gitignore`.